keyboards/tn_alice/rules.mk

```diff
@@ -0,0 +1,10 @@
+# Wildcard to allow APM32 MCU
+DFU_SUFFIX_ARGS = -p FFFF -v FFFF
+
+# WS2812 DRIVER
+WS2812_DRIVER = pwm
+
+RGBLIGHT_ENABLE = yes

```

Comment:
```suggestion
```

Reply:
Hi @zvecr, if I delete this line, does the RGB light still work?